config/resources/document-domain.lisp

```diff
@@ -40,6 +40,9 @@
             (meeting                    :via ,(s-prefix "ext:zittingDocumentversie")
                                         :inverse t
                                         :as "meeting")
+            (meeting                    :via ,(s-prefix "besluitvorming:heeftNotulen")
+                                        :inverse t
+                                        :as "meeting")
```

Comment:
Not sure if having the same key as the above one is a good idea, although I assume they will never be present at the same time?

Reply:
Hmm... depends if we still want the minutes to show up on the documents page of the meeting (`/vergadering/<meeting-id>/agenda/<agenda-id>/documenten`), which I suspect not.

Also see the frontend possibly going crazy because of the `has-one`/`has-many` conflict.
https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/e660ce9a47092a46b264d2d824b0b876c51f09c2/app/models/meeting.js#L36-L40
https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/e660ce9a47092a46b264d2d824b0b876c51f09c2/app/models/meeting.js#L44

Would give this a different name to be safe... Something like `is-minutes-for-meeting` ? (feel free to think of a better name)

Reply:
yeah ... very likely this is going to make something go haywire ... would definitely change that. Naming is hard. Something like `meeting-from-piece`?